hivemind/modules/api/migrations/0004_auto_20181003_1015.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.1.1 on 2018-10-03 08:15
+
```

Comment:
Not quite comfortable with having multiple Migrations for a new feature as it hasn't been implemented on any thing yet. How about we have one migration for this new model ?